app/controllers/PaymentController.scala

```diff
@@ -24,39 +25,40 @@ class PaymentController @Inject()(ws:WSClient,cc: ControllerComponents) extends
       val accessToken: String = (json \ "access_token").as[String]
       accessToken
     }
+  def createOrder(json:JsValue) ={
+    val token:String="Bearer "+getAccessToken()
+      Await.result(ws.url("https://api.sandbox.paypal.com/v2/checkout/orders").addHttpHeaders("Content-Type"->"application/json").addHttpHeaders("Authorization"->token).post(json),Duration(5,"seconds")).json
+  }
+  def capturePayment(orderID:String)=Action{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method takes a value that is controlled by the client](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5261906)